src/applications/vaos/actions/newAppointment.js

```diff
@@ -254,7 +254,9 @@ export function openFacilityPage(page, uiSchema, schema) {
         parentId = parentFacilities[0].institutionCode;
       }
 
-      const systemId = getSystemFromParent(getState(), parentId);
+      const systemId = parentFacilities?.find(
```

Comment:
I might be missing something, but isn't this doing the same thing as `getSystemFromParent()`?

Reply:
Never mind I think I understand. The previous state we were passing in did not have the latest parentFacilities.  LGTM!